src/main/java/org/jabref/gui/groups/GroupTreeViewModel.java

```diff
@@ -372,12 +377,42 @@ public void editGroup(GroupNodeViewModel oldGroup) {
 
                 dialogService.notify(Localization.lang("Modified group \"%0\".", group.getName()));
                 writeGroupChangesToMetaData();
-                // This is ugly but we have no proper update mechanism in place to propagate the changes, so redraw everything
+                // This is ugly, but we have no proper update mechanism in place to propagate the changes, so redraw everything
                 refresh();
             });
         });
     }
 
+    public void chatWithGroup(GroupNodeViewModel group) {
+        // This should probably be done some other way. Please don't blame, it's just a thing to make it quick and fast.
```

Comment:
This branch should never happen? --> `LOGGER.error`?

Reply:
I don't know when it happens. But probably on some super special conditions.

Otherwise, we can't proceed if there is no `currentDatabase`